src/routes/(app)/namespaces/[namespace]/workflows/+page.svelte

```diff
@@ -6,6 +6,14 @@
 
   import AdvancedVisibilityGuard from '$lib/components/advanced-visibility-guard.svelte';
   import WorkflowsWithNewSearch from '$lib/pages/workflows-with-new-search.svelte';
+
+  const columns = [
```

Comment:
I've temporarily added this `columns` variable so that we can calculate `colspan` for the tables.